jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceImplTest.java

```diff
@@ -120,6 +125,8 @@ public void cleanup() {
                 deploymentService.undeploy(unit);
             	} catch (Exception e) {
             		// do nothing in case of some failed tests to avoid next test to fail as well
+                    logger.error("---------------------------" + e.getLocalizedMessage());
+                    logger.error("--do nothing in case of some failed tests to avoid next test to fail as well-------------------------" + unit.getIdentifier());
```

Comment:
Do we need this logging? Maybe we can at least decrease the logging level to debug so it doesn't pollute logs when tests are run.

Reply:
My debug code, I removed it.